Update README.md

TBar123 opened the pull request:
Added that OCMS toolkit must be installed and that input files need to be in place before running the config step

TBar123 committed:
Update README.md

Added that OCMS toolkit must be installed and that input files need to be in place before running the config step

nickilott reviewed (changes requested):
The config issue was fixed in #92 and so if you could remove the sentence aobut needing files in input.dir that would be great. Then good to go!

TBar123 committed:
Update README.md

Remove the line about needing the input dir to be in place before running config
